statistics/integration_test.go

```diff
@@ -667,11 +667,7 @@ func TestShowHistogramsLoadStatus(t *testing.T) {
 	require.NoError(t, h.Update(dom.InfoSchema()))
 	rows := tk.MustQuery("show stats_histograms where db_name = 'test' and table_name = 't'").Rows()
 	for _, row := range rows {
-		if row[3] == "a" || row[3] == "idx" {
-			require.Equal(t, "allLoaded", row[10].(string))
-		} else {
-			require.Equal(t, "allEvicted", row[10].(string))
-		}
```

Comment:
After turning on `lite-init-stats` by default, neither pk stats nor index stats are loaded into stats cache when lease is not zero.